ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -366,7 +397,38 @@ module.exports = function CesiumMap(
         },
       })
     },
-    zoomToExtent: function(coords) {},
+    zoomOut: function(opts, done) {},
+    doPanZoom: function(coords, opts = {}) {
+      // debugger
+      const viewRect = createRectangleAroundCoords(coords)
```

Comment:
Coulnd't we call this.panToRectangle at this point?  And I would update this to be called panToCoords